### PR TITLE
Implement low-pass filter for estimated friction torque

### PR DESCRIPTION
**Pull request by isorrentino**
This PR implements:
- a low-pass filter for filtering the estimated friction torque
- a saturation for the estimated friction torque

### CODE REVIEW

**Comment by GiulioRomualdi** [comment]
Hi @isorrentino can you remove the white space?

**Comment by isorrentino** [comment]
> Hi @isorrentino can you remove the white space?

Done

File: devices/JointTorqueControlDevice/include/BipedalLocomotion/JointTorqueControlDevice.h [hunk 1]
```diff
@@ -66,13 +67,15 @@ struct MotorTorqueCurrentParameters
     double kp; /**< proportional gain */
     double maxCurr; /**< maximum current */
     std::string frictionModel; ///< friction model
+    double max_output_friction; /**< maximum output of the friction model */
```
Comment (GiulioRomualdi) [inline devices/JointTorqueControlDevice/include/BipedalLocomotion/JointTorqueControlDevice.h:70]: Can you use the style of the remaining part of the code?

**Review by GiulioRomualdi — CHANGES_REQUESTED** [review]

**Comment by GiulioRomualdi** [comment]
https://github.com/ami-iit/bipedal-locomotion-framework/pull/892/commits/9afee9a4aa01ead5dd0ff404fe9001a3c846f4c9 sob. I've fixed the changelog manually :(

**Review by GiulioRomualdi — APPROVED** [review]

**Comment by isorrentino** [comment]
Hi @GiulioRomualdi I think the failure is not related to this PR

```
99% tests passed, 1 tests failed out of 149

Total Test time (real) = 630.17 sec

Errors while running CTest
The following tests FAILED:
	120 - memcheck_ArucoDetectorTestUnitTests (Failed)
Error: Process completed with exit code 8.
```

**Pull request merged**